pydantic/_internal/_validators.py

```diff
@@ -45,34 +45,70 @@ def sequence_validator(
         return value_type(v_list)  # type: ignore[call-arg]
 
 
-def import_string(value: Any) -> Any:
+def import_string(value: Any, require_colon_for_attributes: bool = False) -> Any:
     if isinstance(value, str):
         try:
-            return _import_string_logic(value)
+            return _import_string_logic(value, require_colon_for_attributes)
         except ImportError as e:
             raise PydanticCustomError('import_error', 'Invalid python path: {error}', {'error': str(e)})
     else:
         # otherwise we just return the value and let the next validator do the rest of the work
         return value
 
 
-def _import_string_logic(dotted_path: str) -> Any:
+def _import_string_logic(dotted_path: str, require_colon_for_attributes: bool) -> Any:
```

Comment:
Question: What happens if we call the function like `_import_string_logic('collections.abc:Mapping', False)`?
It's returning `<class 'collections.abc.Mapping'>`. Is it correct according to `require_colon_for_attributes=False`

Reply:
I've removed this flag and just automatically fallback to `a.b:c` if you try to use `a.b.c` and that module doesn't exist.